### ISSUE TITLE
Define tasks to be done

### ISSUE THREAD

**Issue by Idrinth** [Ardenian/MyInformationCrawler#2]
Currently your readme is extremely vague about what the crawler is meant to do. 

> Define what information should be extracted from that website

This needs to be more exact about what it does extract. Aim for something simple first, like extracting links, link targets and their respective texts.
The way it's worded at the moment sounds like you want to invest years into getting a proof of concept up and running.

**Comment by Ardenian** [comment]
Okay, I had some thoughts about and I cut it down to the basic webcrawling for now. 
There was a mistake from my side, the example of reading out one Wikipedia entry is rather redundant, since it is the same as opening the page in your browser. 

I assume a webcrawler needs to be told what it shall do. Therefore, I imagined users could add rules to it, like what the webcrawler should do ( that's the algorithm term I used in the ReadMe). 

Example: The webcrawler should collect ratings of one object ( ratings, statistics, ...) from different websites. The user can now define rules about what sites to search, the exact location of the information and what should be done with it. I imagine it to be one file. Then, with time passes by, each website has a profile with information ( could be a foulder). As for the capacity, I plan to limit these profiles to a specific kind of topic, to limit the total amount of profiles needed. Of course, a user could extend the total topics and add new ones, but that's for the future. 

Task List
- [ ] Create basic structure, meaning, create the very basics of the webcrawler ( target something on a website and display it)
- [ ] Save rules for searching in files, for each website. I expect the rules to be similar for different websites ( Example: Give out rating of movie XYZ on website A, give out rating of movie XYZ on website B -> This would be two files, one specifying where the rating of movie entries is located in website A, one for rating of movie entries is located in website B, each in one foulder for the website)
